### ISSUE TITLE
[Accessibility] Tab Navigation does not work for chips and multi-select chip component

### ISSUE THREAD

**Issue by prasadkhegade** [primefaces/primeng#11138]
[Accessibility] Tab Navigation does not work for chips and multi-select chip component

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/primeng-chips-demo-eh85ct?file=src%2Fapp%2Fapp.component.ts

**Current behavior**
Accessibility does not work for chips component and hence accessibility user will not be able to use it as tab does not go on close 'x' icon of each single chip.

**Expected behavior**
Tab navigation should work on chip component and focus should go on 'x close icon to delete the entry.

**Minimal reproduction of the problem with instructions**
1. Open the plunkr link
2. Try tab key to navigate on chip to delete it

**What is the motivation / use case for changing the behavior?**
Accessibility user will be able to use it.

**Please tell us about your environment:**
Chrome Version 97.0.4692.99 (Official Build) (64-bit)

* **Angular version:** 5.X
11.2.13

* **PrimeNG version:** 5.X
11.4.5

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
All
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
All

* **Node (for AoT issues):** `node --version` =   
v14.17.2

**Comment by yigitfindikli** [comment]
All accessibility issues will be fixed in 2022 Q2. Thanks for your report.
